### ISSUE TITLE
Future of Coding Weekly 2021/10 Week 1

### ISSUE THREAD

**Issue by marianoguerra** [marianoguerra/future-of-coding-weekly#98]
- [x] Change Month and Week Number
- [x] focWeekExport 2021-04-19 2021-04-26
- [ ] Summary
- [ ] Hashtags
- [x] Check that comment links work (push weekly dump with channel summaries)
- [x] Check to mention right person for moved messages
- [x] Update Search Index
- [x] Download New Attachments

- https://tinyletter.com/
- https://tinyletter.com/marianoguerra/letters/
- http://localhost:8000/history/
- https://marianoguerra.github.io/future-of-coding-weekly/
- https://stackedit.io/

**Comment by marianoguerra** [comment]
💡 Subtext 10 & Flourish Lang  📜 New Kind of Paper & Papers 🧭 Code Navigation 🕶️ VR & Work

# Two Minute Week

🎥 [Quick demo of Sturdy with an example project](https://www.youtube.com/watch?v=D54x6wBRCH0) via **Kiril Videlov**

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/two-minute-week.html#2021-09-30T08:34:04.006Z)

![Thumbnail](https://img.youtube.com/vi/D54x6wBRCH0/hqdefault.jpg)

We are a tiny team building real-time version control called Sturdy. Here is a quick demo video I recorded this week for the first time.

The main feature I am showcasing is how easy it is to try each other’s work in progress code. Would love to hear what you think. 🙂

**Comment by marianoguerra** [comment]
# Our Work

💡 [github.com/FlourishLang/flourish1.5](https://github.com/FlourishLang/flourish1.5) via **Shakthi Prasad GS**

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/share-your-work.html#2021-09-27T04:42:13.043Z)

Hi Every one,

I am working on an interactive programming language named Flourish,  which aims to be programmable only in run time. The only available programming style being exploratory programming, I believe it implicitly forces TDD.  Even the concept of dependency injection is somewhat inclusive of language design.


Please give your thoughts.

**Comment by marianoguerra** [comment]
# Thinking Together

💬 [Rob Haisfield](https://twitter.com/RobertHaisfield)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/thinking-together.html#2021-09-28T01:35:56.197Z)

Is there some name for the combination of text files and an IDE? A grouping like “GUI”

**Comment by marianoguerra** [comment]
💬 [Ivan Reese](https://twitter.com/spiralganglion)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/thinking-together.html#2021-09-30T22:35:11.240Z)

Did programming go through the "PC revolution" too? Because to me, programming pre-PC, during PC, and now post-PC (tablet, etc) seems to have changed far less than hardware and other software, especially with respect to the qualitative nature of the changes the PC revolution brought about.



Change my mind? Anyone else ever talk about this with this lens?

**Comment by marianoguerra** [comment]
💬 [Mariano Guerra](https://twitter.com/warianoguerra)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/thinking-together.html#2021-10-01T10:01:14.244Z)

Which tools/processes/media do you use to "approach" a problem before starting to write code in a text editor/IDE?

**Comment by marianoguerra** [comment]
# Content

💡 [Subtext 10 by Jonathan Edwards](https://github.com/JonathanMEdwards/subtext10/blob/master/doc/language.md) via [Kartik Agaram](https://github.com/akkartik/mu)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-09-27T06:53:00.132Z)

Subtext 10 by [Jonathan Edwards](https://twitter.com/jonathoda/)

**Comment by marianoguerra** [comment]
🕶️ [Simular VR](https://simulavr.com/) via [Scott Anderson](https://twitter.com/ImpossibleScott)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-09-29T16:26:36.136Z)

For those of you that want an HMD with no Facebook attachment that is designed for productivity

**Comment by marianoguerra** [comment]
🕶️ [Working From Orbit](https://blog.immersed.team/working-from-orbit-39bf95a6d385) via **David Brooks**

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-09-29T16:37:39.137Z)

along those lines [Scott Anderson](https://twitter.com/ImpossibleScott) something I ran across the other day

**Comment by marianoguerra** [comment]
📝 [A New Medium for Communicating Research on Programming Languages](https://willcrichton.net/nota/) via [Srini Kadamati](https://twitter.com/SriniKadamati)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-09-29T19:16:09.139Z)

Just found [nota](https://willcrichton.net/nota/) by [Will Crichton](willcrichton.net) super cool! 👋

**Comment by marianoguerra** [comment]
🕹️ [Fancade: Unlock a world of games! Or make your own?](https://play.fancade.com/) via [Mariano Guerra](https://twitter.com/warianoguerra)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-09-30T09:26:02.140Z)

Fancade has a web version now

**Comment by marianoguerra** [comment]
📜 [New Kind of Paper](https://mlajtos.mu/posts/new-kind-of-paper) via [Mariano Guerra](https://twitter.com/warianoguerra)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-10-01T10:22:08.143Z)

**Comment by marianoguerra** [comment]
💻 [Anytype — operating system for life](https://anytype.io/en) via [Chris Knott](https://github.com/ChrisKnott)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-10-01T20:52:04.151Z)

Has anyone checked out Anytype at all?



I think I must have signed up for an email list a while ago; just got an alert about getting access to the alpha.



Very little info on the website (which is almost off-puttingly professional for what is apparently FOSS).

**Comment by marianoguerra** [comment]
🧭 [CodeRibbon: The second iteration of the Patchworks editor](https://utk-se.github.io/CodeRibbon/) via [Florian Schulz](https://twitter.com/getflourish)

[🧵 conversation](https://marianoguerra.github.io/future-of-coding-weekly/history/weekly/2021/10/W1/linking-together.html#2021-10-02T07:52:27.156Z)


> TL;DR: Developers spend a lot of time navigating code, so I spent the last 8 years building tools to make code navigation more efficient. I’ll share some design principles that we learned along the way. If you want to try out our latest tool, my student recently released an open source Atom plugin, CodeRibbon



Idea/Concept by [@austinzhenley](https://twitter.com/austinzhenley)

→ [Blog Post](https://web.eecs.utk.edu/~azh/blog/coderibbon.html)

Implementation/Plugin by [@robobenklein](https://twitter.com/robobenklein)

**Comment by marianoguerra** [comment]
https://tinyletter.com/marianoguerra/letters/future-of-coding-weekly-2021-10-week-1
